modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -92,15 +100,27 @@ def _read(cls, filepath_or_buffer, **kwargs):
                 # need python3.7 to .seek and .tell ZipExtFile
                 kwargs["compression"] = compression_type
             else:
-                return cls.single_worker_read(filepath_or_buffer, **kwargs)
+                return cls.single_worker_read(
+                    filepath_or_buffer,
+                    reason=f"Unsupported compression type {compression_type}",
```

Comment:
Could we add something to this string that specifies what compression types are supported, since we do have some version dependent logic here?